viewing_party/party.py

```diff
@@ -1,23 +1,205 @@
 # ------------- WAVE 1 --------------------
+from tests.test_constants import *
+def create_movie(movie_title, genre, rating):
+    if movie_title == None or genre == None or rating == None:
+        return None
+    else:
+        new_movie = {"title":MOVIE_TITLE_1,"genre":GENRE_1,"rating":RATING_1}

```

Comment:
This will pass the test, but it is not a working solution.  This will return the _same_ movie every time create_movie is called.  This function is intended to return a movie based on the variables that are being passed into the function.
```suggestion
        new_movie = {"title":movie_title, "genre":genre, "rating":rating}

```